go-apps/meep-wais/api/swagger.yaml

```diff
@@ -3031,28 +3031,6 @@ components:
           $ref: '#/components/schemas/VhtCapabilities'
       type: object
```

Comment:
_Details_ section should be replaced with _Notes_ section from MEC Sandbox API files.

Reply:
as per above